config/crd/bases/ottoscaler.io_policyrecommendations.yaml

```diff
@@ -61,6 +61,8 @@ spec:
                     properties:
                       id:
                         type: string
+                      isDefault:
```

Comment:
This policy field in the PolicyRecommendation can be a string. I have made the corresponding changes in my fork. This can be left as is.

Reply:
Why would isDefault be a string?